src/ui/server/test-mocks/index.ts

```diff
@@ -117,8 +117,12 @@ const mockRes = () => {
   return res;
 };
 
+const mockErrorMessage = 'Mock error';
+
 const mockSpyPromise = () => jest.fn().mockResolvedValue({});
 
+const mockSpyPromiseRejection = jest.fn(() => Promise.reject(new Error(mockErrorMessage)));
```

Comment:
As above, this can be shortened to `jest.fn().mockRejectedValue(new Error(mockErrorMessage))`

Reply:
Thanks @JoshBinns2000 sorry I forgot about this simplification, had copied from the original tests. Updated 👍